ldapdb/backends/ldap/base.py

```diff
@@ -180,6 +181,7 @@ def __init__(self, *args, **kwargs):
         self.validation = DatabaseValidation(self)
         self.settings_dict['SUPPORTS_TRANSACTIONS'] = True
         self.autocommit = True
+        self.batch_size = None
```

Comment:
Actually, I believe the `batch_size` is database dependent, so it's only defined in the Django's `settings.DATABASES`. At the first place I wondering if a per-query `batch_size` would be helpfull, but as long as the database does not support it, or it doesn't allow enough results (`ldap.SIZELIMIT_EXCEEDED`), we might not give the choice to override it in a query.
It's different for the `TIMEOUT` argument I guess.